nnvm/tests/python/frontend/keras/test_forward.py

```diff
@@ -45,7 +51,7 @@ def get_tvm_output(xs, target, ctx, dtype='float32'):
     xs = [np.random.uniform(size=shape, low=-1.0, high=1.0) for shape in in_shapes]
     keras_out = get_keras_output(xs)
     for target, ctx in ctx_list():
-        tvm_out = get_tvm_output([x.transpose([0,3,1,2]) for x in xs], target, ctx)
+        tvm_out = get_tvm_output([x.transpose([0,3,1,2]) for x in xs ] if len(xs[0].shape) == 4 else xs, target, ctx)
```

Comment:
This is tricky.  If we support 1D Keras operators (e.g. Conv1D), `len(xs[0].shape)` equals to 3 -- this is the same as the LSTM's case.  Let's add a parameter to verify_keras_frontend() to indicate whether we have to convert the input data format.